components/Carousel.js

```diff
@@ -88,28 +90,23 @@ const StyledCarousel = styled(Carousel)`
       font-size: ${rem('48px')} !important;
     }
   }
+`
+
+const StyledArrowButton = styled.button`
+  background-color: transparent;
+  padding: ${extraSmallSpacing};
+  border: none;
+  cursor: pointer;
+  opacity: 0;
+  transition: opacity 300ms;
 
-  .slider-decorator-0, .slider-decorator-1 {
-    opacity: 0;
-    transition: 300ms;
-    background-image: url(/static/images/carousel-arrow.svg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+  img {
     width: ${sliderArrowSize};
     height: ${sliderArrowSize};
-
-    @media (min-width: ${mdBreakpoint}) {
-      top: 50% !important;
-    }
-
-    button {
-      opacity: 0 !important;
-    }
   }
 
-  .slider-decorator-0 {
-    transform: rotateY(180deg) translateY(-50%) !important;
+  :focus {
+    outline: 0;
```

Comment:
This seems terrible accessibility-wise. Can't you add another focus indicator?

Reply:
Added a basic one, but these buttons don't even show until you hover over the component so it doesn't improve things much.

Reply:
Ah, I see... I guess that needs a more thorough solution then (if it's even possible with that carousel plugin). Perhaps we need to live with keyboard / screen reader users only having access to the first carousel page then, as much as it sucks.